elastalert/ruletypes.py

```diff
@@ -160,26 +160,39 @@ class ChangeRule(CompareRule):
 
     def compare(self, event):
         key = hashable(lookup_es_key(event, self.rules['query_key']))
-        val = lookup_es_key(event, self.rules['compare_key'])
-        if not isinstance(val, bool) and not val and self.rules['ignore_null']:
-            return False
-        changed = False
+        values = []
+        elastalert_logger.info(" Inside Change Rule " + str(self.occurrences))
+        for val in self.rules['compare_key'].split(","):
+            lookup_value = lookup_es_key(event, val)
+            values.append(lookup_value)
+        elastalert_logger.info("  " + str(values) + " " + str(values))
+        for val in values:
+            if not isinstance(val, bool) and not val and self.rules['ignore_null']:
```

Comment:
I feel like the `ignore_null` thing could use some more thought (possibly it already could have used some more thought, but particularly with the introduction of a compound key). Right now it's applying to _any_ field, but I might conceptually want to ignore null for one of the fields, but not the other(s).